lib/tasks/task.rake

```diff
@@ -0,0 +1,214 @@
+namespace :allrecipe do
+  task :init, [:sFrequency] => :environment do |task, args|
+    get_ar_url(args.sFrequency.to_i)
```

Comment:
Having a method/task that is empty and directly calls another method isn't refactoring, you're kind of wasting your time...